editoast/editoast_schemas/src/rolling_stock/mod.rs

```diff
@@ -0,0 +1,2 @@
+mod gamma;
+pub use gamma::*;
```

Comment:
I tend to avoid `*` in `use` since it might expose new API without us noticing. Also, it's more explicit for future readers of this code (with `*`, they need to go and scan the whole `gamma.rs` file to see what is `pub`).